tests/connect/test_basic_column.py

```diff
@@ -0,0 +1,38 @@
+from __future__ import annotations
+
+from pyspark.sql.functions import col
+from pyspark.sql.types import StringType
+
+
+def test_column_operations(spark_session):
+    # Create DataFrame from range(10)
+    df = spark_session.range(10)
+
+    # Test __getattr__
+    # todo: https://github.com/Eventual-Inc/Daft/issues/3433
+    # df_attr = df.select(col("id").desc())  # Fix: call desc() as method
```

Comment:
This is pretty interesting... I'm not sure if we can support this in Daft today. We're right to bail on this.

What's the expected behavior if a user calls `df.select(col("id").desc(), col("id"))` in Spark?

Reply:
hmmmm good question... 

https://spark.apache.org/docs/latest/api/python/reference/pyspark.sql/api/pyspark.sql.Column.desc.html#pyspark.sql.Column.desc